pkg/apis/extensions/v1beta1/types.go

```diff
@@ -194,7 +194,7 @@ type DeploymentSpec struct {
 	// zero and not specified. Defaults to 1.
 	Replicas *int32 `json:"replicas,omitempty"`
 
-	// Label selector for pods. Existing ReplicationControllers whose pods are
+	// Label selector for pods. Existing ReplicaSets whose pods are
```

Comment:
Done. Sorry about that. This is the second time you are making this comment. These things somehow keep sneaking in.